### PR TITLE
Added Sidepanel Routing

### PR DESCRIPTION
**Pull request by sanjam-deriv**

### CODE REVIEW

File: App.jsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { Router,Outlet} from '@tanstack/react-location';
 import { routes,location } from './Router';
 import { Suspense } from 'react';
 import Header from './components/Header/Header';
+import Docs from './Docs';
```
Comment (adam-deriv) [inline App.jsx:7]: where is this used? I don't see in this file

File: App.jsx [hunk 2]
```diff
@@ -17,6 +18,7 @@ function App() {
           </Router>
       </QueryClientProvider>
     </Suspense>
+
```
Comment (adam-deriv) [inline App.jsx:21]: remove empty line

File: Docs.jsx [hunk 3]
```diff
@@ -1,8 +1,21 @@
...
+import { Suspense } from 'react';
+import Header from './components/Header/Header';
+import Sidepanel from './Sidepanel';
+
+
+function Docs() {
+  return (
+    <Suspense fallback={<div>loading</div>}>
+      <Sidepanel>
+        <Router rouutes={routes} location={location}>
```
Comment (adam-deriv) [inline Docs.jsx:13]: is it 'rouutes'? Maybe it is a typo? Can you check?

File: Sidepanel.jsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+//         <div>
+//              <Link to="/">Home</Link>
+//              <Link to="docs">Quickstart</Link>
+//              <Link to="api_explorer">API Explorer</Link>
+//              <Link to="app_registartaion">App registartaion</Link>
+//              <Link to="api_guide">API guide</Link>
+                   
+//         </div>
+//     );
+// };
```
Comment (adam-deriv) [inline Sidepanel.jsx:37]: remove this commented part?

**Review by adam-deriv — CHANGES_REQUESTED** [review]

File: Docs.jsx (hunk 3 above)
Comment (sanjam-deriv) [reply Docs.jsx]: Yes, this was a typo

File: App.jsx (hunk 1 above)
Comment (sanjam-deriv) [reply App.jsx]: Not needed so removed

File: Docs.jsx [hunk 5]
```diff
@@ -1,8 +1,20 @@
-export default function Docs(){
-    return (
-        <div>
-          I am an "expensive" component... which really just means that I was
-          code-split 😉
-        </div>
-    )
-}
\ No newline at end of file
+import './stateSignal';
```
Comment (adam-deriv) [inline Docs.jsx:1]: do you need this line? This is only to be initialised once where our App.jsx loads

File: Sidepanel.jsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+
+  )
+}
+
+export default Sidepanel
+
+  
+  
+
```
Comment (adam-deriv) [inline Sidepanel.jsx:28]: remove 26-28 empty lines

**Review by adam-deriv — CHANGES_REQUESTED** [review]

**Review by adam-deriv — APPROVED** [review]

**Pull request merged**